amd_build/hip_files/THC/THCApply.cuh.hip

```diff
@@ -0,0 +1,878 @@
+#ifndef THC_APPLY_INC
+#define THC_APPLY_INC
```

Comment:
Same here: it would be good to know (via a comment at the top of the file), what the substantive differences of this file and `THCApply.cuh` are.